addons/udes_stock/models/stock_picking.py

```diff
@@ -871,8 +871,12 @@ def validate_picking(self, create_backorder=False, force_validate=False):
             # Check if a backorder is required and can be created:
             # a) Multi users is enabled
             # b) There are incomplete move lines and create backorder is allowed
+            # c) There are incomplete moves (ie partially available) and create backorder is allowed
             multi_users_enabled = self.picking_type_id.u_multi_users_enabled
-            if self.move_line_ids.get_lines_incomplete():
+            if (
+                self.move_line_ids.get_lines_incomplete()
+                or self.move_lines.get_moves_with_quantity_todo()
```

Comment:
Wouldn't we only need get_moves_with_quantity_todo()  ?

Reply:
Fair point, if we have a move line in a picking and not the move then something elsewhere has gone very wrong and we're all going to be very sad. Have made the change.